libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -689,17 +814,37 @@ type MessagingAPI =
         :> ZConn
         :> "conversations"
         :> Capture "cnv" ConvId
-        :> QueryParam "ignore_missing" IgnoreMissing
-        :> QueryParam "report_missing" ReportMissing
         :> "otr"
         :> "messages"
-        :> ReqBody '[Servant.JSON, Proto] NewOtrMessage
+        :> QueryParam "ignore_missing" IgnoreMissing
+        :> QueryParam "report_missing" ReportMissing
+        :> ReqBody '[JSON, Proto] NewOtrMessage
         :> MultiVerb
              'POST
              '[Servant.JSON]
              (PostOtrResponses ClientMismatch)
-             (Either (MessageNotSent ClientMismatch) ClientMismatch)
+             (PostOtrResponse ClientMismatch)
     )
+    :<|> Named
+           "post-otr-broadcast-unqualified"
```

Comment:
Is the former `Error.nonBindingTeam` missing by intention?

Reply:
The error is still here, but now, unlike before, it is tracked via types, hence not manually written via `CanThrow`. If you check the signature of `Galley.API.Update.postOtrBroadcastUnqualified`, which is the handler for this endpoint, you'll see that it has `Error TeamError` as an effect constraint. The team store action combinator you brought up earlier (now called `lookupBindingTeam`) throws a `NotABindingTeamMember`, which in `Wire.API.Error` gets mapped to `nonBindingTeam`:

```haskell
instance APIError TeamError where                                                                                 
...
  toWai NotABindingTeamMember = nonBindingTeam
...
```